protocol/message.go

```diff
@@ -216,6 +262,40 @@ func NewMonitoringProof(ap []*m.AuthenticationPath,
 	}, ReqSuccess
 }
 
+// NewObservedSTR creates the response message a CONIKS auditor
+// sends to a client, or a CONIKS directory sends to an auditor,
```

Comment:
I think in either case, pub-sub or request-response, the implementation of the corresponding auditor-directory message would be the same, so it would make sense to use the same function to return a list of requested STRs. Similarly, the response message type for a client-auditor request and an auditor-server request for STRs should be the same type. This being said, since this PR is only for client-auditor communication, I won't add relevant auditor-directory documentation yet.